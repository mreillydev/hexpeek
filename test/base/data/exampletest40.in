811 
i 44
