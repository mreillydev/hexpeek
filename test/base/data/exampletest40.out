0000000000000811: 11223344 55667788 99aabbcc ddeeff00  ."3DUfw.........
